webgrid/extensions.py

```diff
@@ -418,3 +418,7 @@ def get_args(self, grid):
             args = loader(self).get_args(grid, args)
 
         return args
+
+    def request_context(self, url='/'):
```

Comment:
The docstring specifies tests, but we should call the method itself `test_request_context`.